website/content/docs/upgrading/index.mdx

```diff
@@ -115,7 +115,7 @@ in the future.
 2. Check the help text from running `waypoint server upgrade -help`.
 
 3. Ensure you are in the right server context before upgrading. This can be
-   verified through `waypoint context`.
+   verified through `waypoint context verify`.
```

Comment:
The original wording here is confusing, so we should clarify what the step is trying to tackle.  I wrote it so that's on me 😅 

The idea here was to look at which context you are currently using to make sure you are going to be upgrading to the right server before invoking the upgrade command (since you could have multiple valid contexts and server installs).

So this step should really say that you'll want to `waypoint context inspect` to make sure you're on the right context intended to be upgraded, and if not, do a `waypoint context list` to find the right one and switch to it using `waypoint context use`. You could end it by doing a `waypoint context verify` although the upgrade command does this too before it attempts to upgrade.

Reply:
Ok, yeah I thought about putting `context inspect`, but `context verify` also returns the context it tried to verify, so I figured that was two birds with one stone and more user-friendly. Changed it to be more explicit.